.github/workflows/docker-ci.yml

```diff
@@ -75,3 +82,34 @@ jobs:
         run: |
           docker load --input /tmp/pysages.tar
           docker run -t pysages bash -c "cd PySAGES/examples/abf/ && python3 ./alanine-dipeptide_openmm.py"
+
+  umbrella-integration-hoomd:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: pysages
+          path: /tmp
+      -
+        name: Load and run test
+        run: |
+          docker load --input /tmp/pysages.tar
+          docker run -v /tmp:/tmp -t pysages bash -c "cd PySAGES/examples/umbrella_integration/ && python3 ./gen_gsd.py && python3 integration.py --N-replica=5 --time-steps=1000 --discard-equi=0 && cp hist.pdf /tmp/hist.pdf && cp energy.pdf /tmp/energy.pdf"
+      -
+        name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: integration-hist.pdf
+          path: /tmp/hist.pdf
+      -
+        name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: integration-energy.pdf
+          path: /tmp/energy.pdf
```

Comment:
Can these be uploaded together. Is there a way to glob files like `/tmp/*.pdf`?

Reply:
the answer is probably yes.
I don't know how yet. But I could try to look it up if it is important.

Reply:
Not that important, just wondering in case was easy to do/figure out.